fms/models/hf/gpt_bigcode/__init__.py

```diff
@@ -0,0 +1,91 @@
+import os
+from typing import Union
+
+from fms.models.hf.gpt_bigcode.modeling_gpt_bigcode_hf import (
+    HFAdaptedGPTBigCodeForCausalLM,
+)
+
+
+def get_model(
```

Comment:
I kept it get_model here as we have the same function for llama in models/hf/llama/__init__.py. I think for now we can keep the name,  but it will eventually be removed by the model loading api.